### ISSUE TITLE
1.15.2: Color nicknames not showing in global chat

### ISSUE THREAD

**Issue by NerdyBirder** [kangarko/ChatControl-Pro#1853]
Chatcontrol version: 8.7.9, Bungeecontrol version: 2.3.3

When players have nicknames using minecraft color codes, the nickname show up as white in global chat. Color nicknames showed up in global chat for everyone before I updated to version 8.7.9.
There are no console errors in bungeecord or any of the bungeecord instances.

[ChatControl(Aug-15-2020).zip](https://github.com/kangarko/ChatControl-Pro/files/5079270/ChatControl.Aug-15-2020.zip)

**Comment by mrcoffee1026** [comment]
Same here, no errors, the nicknames are clearly acquiring the colors if I check their values via other methods they just cannot display.

**Comment by james090500** [comment]
Believe this is related to #1852

**Comment by kangarko** [comment]
Thanks, I will prioritize this in 2-3 days

**Comment by Sikatsu** [comment]
Is this related to the {nick} placeholder, for example would cmi's nick placeholder work fine? Just asking in case anyone knows ^^

**Comment by mrcoffee1026** [comment]
I'm not using bungee so I don't THINK it's related although I can't be entirely certain.  I can't say if it has to do with the {nick} placeholder or not either... there's a couple other random language locations in essentialsx I can get the nick to display with the currently set colors, but in their language file they use something like {0} or {1}, so if it IS a problem with {nick}, essentialsx wouldn't notice.

**Comment by NerdyBirder** [comment]
Yes, we are using the nick placeholder to show nicknames in global chat and that is where the color is lost on my server.

**Comment by mrcoffee1026** [comment]
I'll also be switching to use the %cmi_user_display_name% placeholder instead of {nick} since while this is supposedly fixed now, no new releases have been put up since it has been fixed and so the issue persists.

**Comment by kangarko** [comment]
Thanks, this is getting done however it got stalled due to some features causing deadlock on online servers, rewriting it since Saturday, ETA 1-3 days

**Comment by mrcoffee1026** [comment]
ok cool, thanks!
